ext/openssl/tests/openssl_free_key.phpt

```diff
@@ -0,0 +1,77 @@
+--TEST--
+void openssl_free_key ( resource $key_identifier );
+--CREDITS--
+marcosptf - <marcosptf@yahoo.com.br> - @phpsp - sao paulo - br
+--SKIPIF--
+<?php
+if (!extension_loaded("openssl"))
+    die("skip");
+if (!@openssl_pkey_new())
+    die("skip cannot create private key");
+?>
+--FILE--
+<?php
+echo "Creating private key\n";
+
+/* stack up some entropy; performance is not critical,
+ * and being slow will most likely even help the test.
+ */
+for ($z = "", $i = 0; $i < 1024; $i++) {
+    $z .= $i * $i;
+    if (function_exists("usleep"))
```

Comment:
usleep() is always available, however if HAVE_USLEEP is not available on the system (its always available on Windows btw), then the function simply returns NULL and does not yield anything. So removing the check all together and simply let system that do not have a proper usleep() pass through.